cmd/snap/main.go

```
@@ -92,6 +94,17 @@ type parserSetter interface {
 // Since commands have local state a fresh parser is required to isolate tests
 // from each other.
 func Parser() *flags.Parser {
+	optionsData.Version = func() error {
```

An error is never returned?